build.gradle

```diff
@@ -6,7 +6,7 @@ buildscript {
 
     CONFIG.versions.android.sdk.min = 21
     CONFIG.versions.kotlin = '1.3.61'
-    CONFIG.versions.android.plugin = '3.5.3'
+    CONFIG.versions.android.plugin = '3.6.2'
```

Comment:
I think you can remove this line. `CONFIG.versions.android.plugin` from android-buildconfig is already set to `3.6.2` currently.